research-domain-agent/src/test/java/care/smith/fts/rda/rest/TransferProcessControllerTest.java

```diff
@@ -38,25 +47,25 @@ void setUp() {
             new TransferProcessRunner() {
               @Override
               public String start(TransferProcessDefinition process, Mono<TransportBundle> data) {
-                return "processId";
+                return RUNNING_PROCESS_ID;
               }
 
               @Override
               public Mono<Status> status(String processId) {
-                if ("processId".equals(processId)) {
-                  return Mono.just(new Status("processId", Phase.RUNNING, 0, 0));
-                } else {
-                  return Mono.error(new RuntimeException("error"));
-                }
+                return switch (processId) {
```

Comment:
This looks awful. Can we use Mockito to mock the Runner?

Reply:
I agree that it looks terrible (and this is already the nice version).

We could mock it but then we would test something else...